libraryValidations/Python/test_json_validations.py

```diff
@@ -119,7 +121,5 @@ def run_tests(self, test_key):
                 user = feature_flag_test[INPUTS_KEY].get(USER_KEY, None)
                 groups = feature_flag_test[INPUTS_KEY].get(GROUPS_KEY, [])
                 variant = feature_manager.get_variant(feature_flag_test[FEATURE_FLAG_NAME_KEY], TargetingContext(user_id=user, groups=groups))
-                if not variant:
-                    logger.error(f"Variant is None for {feature_flag_id}")
-                    assert False, failed_description
+
```

Comment:
Is there a reason you removed this assert? If you unindent it and replace `False` with `variant` it should work, see provider version.

Reply:
My goal was to separate null variant from a variant whos configuration value was null. But I just updated this in a different way with my latest commit.